tests/range/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,12 +1,77 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Good call, this uncovered a bug with ranges that don't end on nodes. Fixed in https://github.com/prettier/prettier/pull/1659/commits/6bb85b693423dcd64794050e399019199e8dfdd9 and https://github.com/prettier/prettier/pull/1659/commits/ca6148bb5f1ca408cef635f5fe431251de453593